JavaScript/JavaScript.sublime-syntax

```diff
@@ -8,6 +8,10 @@ file_extensions:
 first_line_match: ^#!\s*/.*\b(node|js)\b
 scope: source.js
 variables:
+  ddigits: (?:[0-9_]+)
+  ddigits0: (?:0|[1-9][0-9_]*)
```

Comment:
The variable names might be a bit unclear.

Reply:
They are borrowed from the other syntaxes which have been recently tweaked.